swagger_parser_pages/lib/content/generator_content.dart

```diff
@@ -268,9 +334,13 @@ Future<void> _generateOutputs(
     language: language,
     outputDirectory: '',
     freezed: freezed,
-    rootInterface: rootInterface,
+    rootClient: rootClient,
+    rootClientName: rootClientName.trim().isEmpty ? null : rootClientName,
     clientPostfix: clientPostfix.trim().isEmpty ? null : clientPostfix,
-    squishClients: squishClients,
+    name: name,
```

Comment:
trim for name

Reply:
https://github.com/Carapacik/swagger_parser/pull/97/commits/8a990f7f235fd3317ec9711d621affa667449309